Tools/AP_Bootloader/board_types.txt

```diff
@@ -248,10 +248,12 @@ AP_HW_PixFlamingoH743V               1133
 
 AP_HW_AR-F407SmartBat                1134
 AP_HW_SPEEDYBEEF4MINI                1135
+<<<<<<< HEAD
```

Comment:
this is a committed merge conflict, needs fixing

Reply:
sorry..fixed